### ISSUE TITLE
HtmlRenderer on OpenGLES

### ISSUE THREAD

**Issue by prepare** [LayoutFarm/HtmlRenderer#55]
(pure hardware rendering1)
Render directly on the OpenGLES surface (without GDI+)
coordinate errors/ temp fix, snapshot.

![html_gles2](https://cloud.githubusercontent.com/assets/7447159/22859298/7426d73c-f109-11e6-9204-451661611021.png)

**Comment by prepare** [comment]
Nov 2018
---
![htmlbox_gles](https://user-images.githubusercontent.com/7447159/49267489-5ea15e80-f48d-11e8-8214-cd19e3697aa8.png)


see more about glyph texture at https://github.com/PaintLab/PixelFarm/issues/3

**Comment by prepare** [comment]
Progression
---

This is a snap shot of the HtmlRenderer that runs on
**GLFW+GLES2+OpenTK**,
**no WinForms and no GDI+**.


![neutral1](https://user-images.githubusercontent.com/7447159/52188038-0d359700-2863-11e9-972c-b488773667e1.png)
_pic 1: 'neutral' branch_

This example is different from the GLES2 example on 
the readme page. The old one in readme page still
need some WinForm supports, but the new one dose not.


Plan=> 
1. clean up (The code still in a 'mess' state, let me clean it up)
2. create .netcore examples (Win, macOS, Linux).

It should not be a long time :)



In short, essential dll dependencies are

    1. OpenTK ( cross-platform => Yes)
